docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -40,94 +46,77 @@ import app from '../server'
 import User from '../models/User'
 ```
 
-### Your server file
-
-When you test endpoints, you don't want to start your app. Supertest does this for us automatically when you pass your app into it.
-
-So, if you're using express for your application, make sure you have a startup file that imports `app` and calls `app.listen`
-
-Here's an example of the [server file](https://github.com/zellwk/ava-mdb-test/blob/master/server.js)
-
-And here's an example of your Mongoose [model](https://github.com/zellwk/ava-mdb-test/blob/master/models/User.js)
-
-### Back to your test file
-
-**First, start your MongoDB instance and connect to Mongoose:**
+Next start the in-memory MongoDB instance and connect to Mongoose:
 
 ```js
-// Start MongoDB Instance
+// Start MongoDB instance
 const mongod = new MongodbMemoryServer()
 
-// Create connection to mongoose before all tests
-test.before(async t => mongoose.connect(await mongod.getConnectionString(), { useMongoClient: true }))
+// Create connection to Mongoose before tests are run
+test.before(async t => {
+	const uri = await mongod.getConnectionString();
+	await mongoose.connect(uri, {useMongoClient: true});
+});
 ```
 
-When you run your first test, MongoDB downloads the latest MongoDB Binaries. It may take a minute. (The download is ~70mb).
-
-**Add fixtures for each test**
+When you run your first test, MongoDB downloads the latest MongoDB binaries. The download is ~70MB so this may take a minute.
 
 You'll want to populate your database with dummy data. Here's an example:
 
 ```js
 test.beforeEach(async t => {
-  const user = new User({
-  	email: 'one@example.com',
-  	name: 'One'
-  })
-  await user.save()
-})
+	const user = new User({
+		email: 'one@example.com',
+		name: 'One'
+	});
+	await user.save();
+});
 ```
 
-**Clear your dummy data after each test**:
+Dummy data should be cleared after each test:
 
 ```js
-// Cleans up database after every test
-test.afterEach.always(async t => await User.remove())
+test.afterEach.always(async t => await User.remove());
```

Comment:
No need for the `t` argument here.

I'd change it to `test.afterEach.always(() => User.remove());`